docs: Add minimum linux kernel version to README (#39)
edgexfoundry-holding/device-usb-camera: README.md

## How does the device service work?

> - The device service ONLY works on Linux system.
> - The Linux kernel version is restricted to v5.10.x.

Reviewer: That is the minimum, not the only minor version.
```suggestion
- The device service ONLY works on Linux with kernel v5.10 or higher.
```

Reviewer 2: @farshidtz it seems someone built the binary failed in 5.11 kernel
https://github.com/edgexfoundry/device-usb-camera/issues/10

Reviewer: Maybe their libc-dev wasn't built against 5.11?

Reviewer: The APIs suppose to be backward compatible. Both Alpine 3.15 (used as the docker image base) and Ubuntu 22.04 (use as snap base) use kernel 5.15 right now, on which we can build and run the application without issues.

Author: Added the suggested change. Thank you @farshidtz.
